ModPatches/Vanilla Arsenal/Defs/Vanilla Arsenal/Ammo.xml

```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Defs>
+
+	<ThingCategoryDef>
+		<defName>Ammo145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<parent>AmmoHighCaliber</parent>
+		<iconPath>UI/Icons/ThingCategories/CaliberHighCaliber</iconPath>
+	</ThingCategoryDef>
+
+	<!-- ==================== AmmoSet ========================== -->
+
+	<CombatExtended.AmmoSetDef>
+		<defName>AmmoSet_145x114mm_Demo</defName>
+		<label>14.5x114mm demolition</label>
+		<ammoTypes>
+			<Ammo_145x114mm_Demo>Bullet_145x114mm_Demo</Ammo_145x114mm_Demo>
+		</ammoTypes>
+	</CombatExtended.AmmoSetDef>
+
+	<!-- ==================== Ammo ========================== -->
+
+	<ThingDef Class="CombatExtended.AmmoDef" ParentName="Ammo145x114mmBase">
+		<defName>Ammo_145x114mm_Demo</defName>
+		<label>14.5x114mm bullet (Demo)</label>
+		<graphicData>
+			<texPath>Things/Ammo/HighCaliber/HE</texPath>
+			<graphicClass>Graphic_StackCount</graphicClass>
+		</graphicData>
+		<statBases>
+			<MarketValue>1.27</MarketValue>
+		</statBases>
+		<ammoClass>Demolition</ammoClass>
+		<cookOffProjectile>Bullet_145x114mm_Demo</cookOffProjectile>
+	</ThingDef>
+
+	<!-- ================== Projectiles ================== -->
+
+	<ThingDef ParentName="Base145x114mmBullet">
+		<defName>Bullet_145x114mm_Demo</defName>
+		<label>14.5x114mm bullet (Demo)</label>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageAmountBase>27</damageAmountBase>
+			<armorPenetrationSharp>18</armorPenetrationSharp>
+			<armorPenetrationBlunt>634</armorPenetrationBlunt>
+			<secondaryDamage>
+				<li>
+					<def>Thermobaric</def>

```

Comment:
The base mod ammo spreads around burning fuel, which this doesn't currently do. As it stands, it's pretty much just a variant of AP-I. Is that an intended change?

Reply:
Yes it was intended, I was told the rifle felt weird to use when it produced burning chemfuel. I wasn't too sure if I should focus on the flavour text and build the weapon like that or based on the feedback I got and the results I saw during testing